.github/workflows/reassurePerformanceTests.yml

```diff
@@ -47,12 +47,12 @@ jobs:
           BASELINE_BRANCH=${BASELINE_BRANCH:="main"}
           git fetch origin "$BASELINE_BRANCH" --no-tags --depth=1
           git switch "$BASELINE_BRANCH"
-          npm install --force
+          npm install --force || (rm -rf node_modules && npm install --force) 
```

Comment:
I saw your comment with the screenshot, but I'm still a bit hazy on what this pipe is doing

Reply:
The way I interpret this... it would be "if npm install --force fails, then delete node_modules and try again". Is that right?

Reply:
yeah, that's correct. and we wrap the 2nd part in parens because if you just had this:

```sh
npm install --force || rm -rf node_modules && npm install --force
```

then then 2nd `npm install --force` would run regardless of whether the first one succeeds:

<img width="762" alt="image" src="https://github.com/user-attachments/assets/b205431f-6b07-4131-abd7-dbf290865334">